akka-actor-typed-tests/src/test/scala/akka/actor/typed/coexistence/UntypedSupervisingTypedSpec.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.coexistence
+import akka.actor.Actor
+import akka.actor.testkit.typed.TestException
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed._
+import akka.actor.typed.coexistence.UntypedSupervisingTypedSpec.{
+  SpawnAnonFromUntyped,
+  SpawnFromUntyped,
+  TypedSpawnedFromUntypedConext,
+  UntypedToTyped
+}
+import akka.testkit.{ AkkaSpec, ImplicitSender, TestProbe }
+import akka.actor.typed.scaladsl.adapter._
+import akka.{ actor => u }
+
+import scala.concurrent.duration._
+
+object ProbedBehavior {
+  def behavior(probe: u.ActorRef): Behavior[String] = {
+    Behaviors
+      .receiveMessage[String] {
+        case "throw" => throw TestException("oh dear")
+      }
+      .receiveSignal {
+        case (_, s) =>
+          probe ! s
+          Behaviors.same
+      }
+
+  }
+}
+
+object UntypedSupervisingTypedSpec {
+
+  case class SpawnFromUntyped(behav: Behavior[String], name: String)
```

Comment:
finals?

Reply:
not too concerned about test case classes